packages/make-styles/src/shorthands/overflow.ts

```diff
@@ -0,0 +1,18 @@
+import { OverflowProperty } from 'csstype';
+import { MakeStyles } from '../types';
+
+/**
+ * A function that implements CSS spec conformant expansion for "gap"
+ *
+ * @example
+ *   overflow('hidden')
+ *   overflow('hidden', 'scroll')
+ *
+ * See https://developer.mozilla.org/en-US/docs/Web/CSS/overflow
+ */
+export function overflow(overflowX: OverflowProperty, overflowY: OverflowProperty = overflowX): MakeStyles {
```

Comment:
is an overflow shorthand needed ? how do you use `overflow` for separate x and y values ?

Reply:
It's used surprisingly frequently in our styles, that's why I added it 😀

> how do you use overflow for separate x and y values ?

```tsx
{
  // Used a lot in styles
  { overflow: 'hidden' },
  // Not really used
  { overflow: 'hidden visible' }
}
```

Reply:
Well today I learned something. I don't think I've ever encountered the separate one :/